README.md

```diff
@@ -59,6 +60,7 @@ jobs:
         asset_name: mything
         tag: ${{ github.ref }}
         overwrite: true
+        promote: true
```

Comment:
Do we need this here? After all, in this scenario in the README, the user won't have a prerelease here. If you want, you can add a new scenario where you first create a prerelease and then promote it.

Reply:
In my case yes, the way I use this action ti to call it once a new prerelease is created manually form the github website, so I don't really use the action with prerelease first. Does it make sense? 
In any case I can jut remove if from this example, it doesn't really matter

Reply:
It's not really required or does it do anything in these examples so let's remove flags that aren't required as otherwise people might think that they need these somehow.

Reply:
done